rajawali/src/main/java/org/rajawali3d/Object3D.java

```diff
@@ -371,13 +371,18 @@ public boolean isContainer() {
 	}
 
 	/**
-	 * Maps screen coordinates to object coordinates
-	 *
-	 * @param x
-	 * @param y
-	 * @param viewportWidth
-	 * @param viewportHeight
-	 * @param eyeZ
+	 * Positions the object in the plane of the screen/viewport (z = 0), with
+     * x and y specified in GL screen coordinates (origin at bottom left);
```

Comment:
The formatting is all over the place right now. That bit in the guidelines is more about making sure that the commit is not filled with a bunch of noise from whitespace changes. 

As a general rule, once you push something to a public repo, you should not rebase it. We arent super concerned about clean commit histories (look at ours, its a mess) mostly make sure they arent superfluous with a bunch of commits like "changing a constant". Try and get that figured out before submitting the PR. Once you submit a PR, if we ask you to change something then its fine to do it as an additional commit.